src/software/ai/hl/stp/play/corner_kick_play_test.cpp

```diff
@@ -17,7 +17,7 @@ class CornerKickPlayTest : public SimulatedPlayTestFixture
 {
 };
 
-TEST_F(CornerKickPlayTest, test_corner_kick_play)
+TEST_F(CornerKickPlayTest, test_corner_kick_play_bottom_left)
```

Comment:
The cost functions/weights haven't been tuned yet for zoned passing so it makes sense that its different.

Also the robot positions are slightly different which can tip the scales one way or the other

![zonez](https://user-images.githubusercontent.com/31548846/113730955-33f2f080-96ad-11eb-98a8-00dfc972c146.gif)

Basically it looks different because it is different/pending tuning

Reply:
I can't tune everything as part of this PR to prevent scope creep, so we need to reach a middle ground here

to prove that it just needs tuning, here is what happens when I change the sigmoid offset from 0.5 to 0.45 in `ratePassShootScore`

![update](https://user-images.githubusercontent.com/31548846/113774474-315bbf80-96dc-11eb-888b-9061a4bfc51f.gif)

There are a lot of numbers like these that need to be understood and changed/tuned, and I can also imagine switching out the underlying sigmoid implementation will require more tuning.

Reply:
OK it's fine then